content/blog/adoptopenjdk-to-join-the-eclipse-foundation/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: AdoptOpenJDK to Join the Eclipse Foundation
+date: "2020-06-19T09:00:00+00:00"
+author: tsc
+description: AdoptOpenJDK will Join the Eclipse Foundation and rebrand as Adoptium.
```

Comment:
...will **j**oin (small j).

I prefer "rename to Adoptium" instead of "rebrand". I'm no native speaker, however.

Reply:
I think rebrand is better, we're doing a full branding change here, Logo etc